stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -419,11 +477,15 @@ static String createQuery(Map<String, Object> params)
         JSONObject headerMappingObject = new JSONObject(propertyMap);
         headers.put("X-Stripe-Client-User-Agent", headerMappingObject.toString());
 
-        if (apiVersion != null) {
-            headers.put("Stripe-Version", apiVersion);
+        if (options != null && options.getApiVersion() != null) {
+            headers.put("Stripe-Version", options.getApiVersion());
         }
 
-        if (options.getIdempotencyKey() != null) {
+        if (options != null && options.getGuid() != null) {
+            headers.put("guid", options.getGuid());
```

Comment:
Does that make it a cookie somehow? I can make the change; will that interfere with the other items around it that aren't in this format?